lago/cmd.py

```diff
@@ -83,6 +83,13 @@
 @lago.plugins.cli.cli_plugin_add_argument(
     '--template-store',
     help='Location to store templates at',
+    default='/var/lib/lago/store',
```

Comment:
is there a way to 'interpolate' the values? I say because it might be nicer to use something like '{base_dir}/store' so you don't have to repeat everywhere and it also makes it easier to change. A quick solution might be to just add the base path as another param and make these others relative.